### DISCUSSION TITLE
Flush tensorboard writer to monitor in real-time

### DISCUSSION THREAD

**Discussion by AmmarRashed** [TorchEnsemble-Community/Ensemble-Pytorch#161] [Ideas]
Hello,
- Unless the writer is flushed explicitly with `tb_logger.flush()`, tensorboard will not show any results until the log file is closed.
- It would also be great to report the loss, not just the accuracy, to consider for imbalanced datasets.
Thanks,

**Comment by xuyxu** [comment]
Hi @AmmarRashed, thanks for your suggestion. Wondering that if you are willing to add this?

**Reply by AmmarRashed** [reply]
Thanks for the reply. I actually tried adding a simple flush, but for some reason the results still don't show on tensorboard. I will be happy to work on and debug it and submit a pull request. Stay tuned :)

**Reply by xuyxu** [reply]
Glad to hear that! Please feel free to comment below if you need any help.
